.circleci/config.yml

```diff
@@ -445,37 +451,15 @@ jobs:
           cf-username: CF_USERNAME_PROD
           frontend-appname: tdp-frontend-prod
 
-  deploy-raft:
-    executor: docker-executor
-    working_directory: ~/tdp-deploy
-    steps:
-      - deploy-cloud-dot-gov:
-          backend-appname: tdp-backend-raft
-          frontend-appname: tdp-frontend-raft
+parameters:
```

Comment:
These are [Pipeline Parameters](https://circleci.com/docs/2.0/pipeline-variables/#pipeline-parameters-in-configuration) - which can be passed to the [Trigger Pipeline](https://circleci.com/docs/api/v2/#operation/triggerPipeline) endpoint in the new API as a JSON payload.